### DISCUSSION TITLE
Does anyone know why nothing shows up in HA until midday?

### DISCUSSION THREAD

**Discussion by damianharbr** [hultenvp/solis-sensor#211] [Q&A] [answered]
I am getting solar every day, but nothing shows up until midday and then the solar keeps working till sunset. 

I can see I am exporting, but nothing is updating from solis till then.

**Comment by hultenvp** [comment]
I suggest you check the timezone settings on the server.
Timezone had an effect on what's being reported and it refuses to send data when it thinks it's dark outside.

**Reply by damianharbr** [reply]
TZ settings definitely correct on the server. Is there anything in any of the recent updates that could have perhaps changed this?

Or maybe on my install didn't work correctly?

**Comment by damianharbr** [comment]
The Timezone settings are correct for London. This has only been happening since the last couple of updates to this repo. 

Around November 9th I think

**Comment by damianharbr** [comment, marked as answer]
I think I have found the issue. 

For some reason the Solis Cloud created two extra phantom inverters alongside the real one. Each with their own Serial Number.

Deleting these and then reinstalling the integration seems to have fixed it!

**Reply by hultenvp** [reply]
Good to hear it is solved (and that's not a bug in the integration 😉  )

**Reply by damianharbr** [reply]
Yeah sorry to bring your integration into doubt!

**Reply by grbasta** [reply]
It continues to show me the solar production starting at 12.00, the time zone is correct, I too have eliminated the integration and reloaded, but the result does not change. Greetings

**Reply by SadGamerGeek** [reply]
Hi. I just came here to report the same thing. Since yesterday my integration seems to be updating everything as expected constantly except Energy Today. Yesterday this was at zero and then I had a big spike where it caught up all the missing energy and reported it somewhere between 12:00 and 13:00, and now this morning 11:58 currently Energy Today is still at zero. The SolisCloud web page is reporting energy as expected. I didn't have a phantom inverter in SolisCloud. Pretty sure this was working OK since I installed it (only about a week ago) until yesterday. TimeZone is correct.

**Reply by SadGamerGeek** [reply]
update - it's now 12:05 and Energy Today has just leapt to 3.4kWh. It definitely didn't use all that since the last update, so once again it looks like all energy before 12:00 is lumped together and reported at 12:00 or just after

**Reply by dhoogfr** [reply]
Noticed this before as well
Restarting HA fixed the issue for me

**Reply by SadGamerGeek** [reply]
Sadly I'd already restarted HA between it happening yesterday and today, so didn't fix mine
